docs/artifacts/two-sessions-one-container.html

```
@@ -0,0 +1,123 @@
+  <div class="eyebrow">claude-code-marketplace · slice 3b · PR #3 · head e77fd7a · 2026-08-28</div>
+  <h1>Two Sessions, One Container</h1>
+  <p class="lede">Until today the CI test typed the install commands itself. Now a Claude agent, handed only the two READMEs, does the installing — and a second, fresh session proves the plugin's start-up hook fires on its own. If the docs have a hole, the PR goes red.</p>
+
+  <h2>What changed, in one picture</h2>
+  <figure>
+    <pre class="mermaid">
+%%{init: {'theme':'base','themeVariables':{'primaryColor':'#e6f2f6','primaryTextColor':'#1c211f','primaryBorderColor':'#0e6f8a','lineColor':'#1c211f','secondaryColor':'#f1f3ef','tertiaryColor':'#fbfcfa','fontFamily':'IBM Plex Sans, sans-serif'}}}%%
+flowchart LR
+  subgraph B["BEFORE — PR #1 (05acca22)"]
+    direction TB
+    s1["script types<br/>marketplace add + plugin install"] --> s2["script runs the hook<br/>by hand (sh -c)"] --> s3["agent runs the CLI"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)